tools/WPA/wpa.cpp

```diff
@@ -59,7 +61,7 @@ int main(int argc, char ** argv) {
 
     SVFModule svfModule(moduleNameVec);
 
-    WPAPass *wpa = new WPAPass();
+    auto wpa = std::unique_ptr<WPAPass>(new WPAPass());
```

Comment:
I would like to avoid using unique_ptr :) It is quite different from the overall coding style of SVF.

Could you please keep it for your own use and revert it back?

Reply:
No problem, will revert. What's SVF's style for this? I saw multiple occurrences of raw new?